packages/sdk-js/src/util.ts

```diff
@@ -62,6 +62,42 @@ export function getUrlRegExp(regexString: string): RegExp | undefined {
   }
 }
 
+export function isURLTargeted(url: string, targets: UrlTarget[]) {
+  if (!targets.length) return true;
+  let hasIncludeRules = false;
+  let isIncluded = false;
+
+  for (let i = 0; i < targets.length; i++) {
+    const match = _evalURLTarget(url, targets[i].type, targets[i].pattern);
+    if (targets[i].include === false) {
+      if (match) return false;
+    } else {
+      hasIncludeRules = true;
+      if (match) isIncluded = true;
+    }
+  }
+
+  return isIncluded || !hasIncludeRules;
+}
+
+function _evalURLTarget(
+  url: string,
+  type: UrlTargetType,
+  pattern: string
+): boolean {
+  const pathOnly = url.replace(/^https?:\/\//, "").replace(/^[^/]*\//, "/");
```

Comment:
probably fine for now, but do we need to worry about query strings (and their order)?

Reply:
same question @bryce-fitzsimons